astroquery/sdss/core.py

```diff
@@ -1072,5 +1075,55 @@ def _get_crossid_url(self, data_release):
         self._last_url = url
         return url
 
+    def view_in_sdss_navigate(self, sc):
+        """
+        To view "Navigate" given the SkyCoord sc of the point of interest.
+
+        Parameters
+        ----------
+        sc: SkyCoord of a point
+
+        Returns
+        -------
+        Response: Open a browser tab in the SDSS "Navigate" interface
+
+        """
+        if isinstance(sc, SkyCoord):
+            if sc.shape == ():
+                payload = {"ra": str(Angle(sc.ra).degree), "dec": str(Angle(sc.dec).degree), "opt": "GO"}
+                r = requests.get("http://skyserver.sdss.org/dr15/en/tools/chart/navi.aspx", params=payload)
```

Comment:
Use the `Query._request` interface for this rather than an independent `requests` query.  It doesn't matter a whole lot in this case, but we want consistency throughout astroquery.

Reply:
Yup, changed to `self.request(...)` in code.